tests/rados/test_add_db_wal_bst.py

```diff
@@ -0,0 +1,138 @@
+from ceph.rados.core_workflows import RadosOrchestrator
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def run(ceph_cluster, **kw):
+    log.info(run.__doc__)
+    config = kw["config"]
+    cephadm = CephAdmin(cluster=ceph_cluster, **config)
+    ceph_nodes = kw.get("ceph_nodes")
+    rados_object = RadosOrchestrator(node=cephadm)
+    bluestore_obj = BluestoreToolWorkflows(node=cephadm)
+    osd_flag_status = 0
+    try:
+        # Get the OSD list
+        for node in ceph_nodes:
+            if node.role == "osd":
+                host_name = node.hostname
+                ssd_path_list = rados_object.get_available_ssd(host_name)
+                if not ssd_path_list:
+                    log.info(f"The device paths are empty to add on the {host_name}")
+                    continue
+                osds_id_in_node = rados_object.collect_osd_daemon_ids(node)
+                log.info(f"The osd's  configured on the HDD are-{osds_id_in_node}")
+                test_osd = random.choice(osds_id_in_node)
+                log.info(f"Performing the configurations on {test_osd}")
+
+                nvme_flag = False
+                ssd_flag = False
+                for dev_path in ssd_path_list:
+                    if "nvme" in dev_path and not nvme_flag:
+                        stop_osd(rados_object, test_osd)
+                        bluestore_obj.add_wal_device(test_osd, dev_path)
+                        start_osd(rados_object, test_osd)
+                        osd_config_result = check_osd_config(
+                            rados_object, host_name, dev_path
+                        )
+                        if not osd_config_result:
+                            log.error(
+                                f"The nvme path {dev_path} is not added to the osd"
+                            )
+                            return 1
+                        nvme_flag = True
+                    elif not ssd_flag:
+                        stop_osd(rados_object, test_osd)
+                        bluestore_obj.add_db_device(test_osd, dev_path)
+                        start_osd(rados_object, test_osd)
+                        osd_config_result = check_osd_config(
+                            rados_object, host_name, dev_path
+                        )
+                        if not osd_config_result:
+                            log.error(
+                                f"The SSD path {dev_path} is not added to the osd"
+                            )
+                            return 1
+                        ssd_flag = True
+
+                if nvme_flag:
+                    log.info(f"The NVMe is configured to the {test_osd}")
+                else:
+                    log.info(f"The NVMe is not configured to the {test_osd}")
+                if ssd_flag:
+                    log.info(f"The SSD is configured to the {test_osd}")
+                else:
+                    log.info(f"The SSD is not configured to the {test_osd}")
```

Comment:
Can we please change the text to indicate which exact device(WAL/DB) has been configured for nvme or SSD

Reply:
Removed the Lines from the code. The log lines are included in the methods.